docs/jwst.rst

```diff
@@ -0,0 +1,254 @@
+.. _jwst_instruments:
+
+
+*****************************
+JWST Instrument Model Details
+*****************************
+
+
+The following describes specific details for the various JWST instrument classes. See also :ref:`the references page <references>` for information on data sources. 
+
+
+One general note is that the webbpsf class interfaces do not attempt to exactly
```

Comment:
When writing about WebbPSF I've been using the capitalized form to refer to the software package and reserving `webbpsf` in lowercase for actual import paths and the like. Not sure if that's something you'd like to continue here.